play-2.4/swagger-play2/app/play/modules/swagger/SwaggerPlugin.scala

```diff
@@ -80,6 +80,24 @@ class SwaggerPluginImpl @Inject()(lifecycle: ApplicationLifecycle, router: Route
     case Some(value)=> value
   }
 
+  // Play seems to throw an exception if this value is not defined and you get it
```

Comment:
@JosephEarl I'm discussing this with the Play devs right now if you'd like to chime in https://gitter.im/playframework/contributors